calico-cloud/operations/cluster-management.mdx

```diff
@@ -8,9 +8,33 @@ description: Manage cluster storage, add new clusters, and configure alerts.
 
 To connect new clusters to {{prodname}}, select Managed Clusters in the left nav, and click **Connect Cluster**.
 
-## Recover a deleted managed cluster
+## Delete a managed cluster
 
-If you inadvertenty delete a managed cluster, rerun the `curl` command and the cluster will reconnect to {{prodname}}.
+### Prerequisites
+
+- Only users with either the `Owner` or `Admin` [roles](../users/user-management.mdx) are permitted to delete managed clusters.
+- The managed cluster must not be connected. For help with disconnecting your managed cluster, see [how to uninstall {{prodname}}](disconnect.mdx).
+
+### Delete a managed cluster
+
+1. In Manager UI, go to **Managed Clusters**.
+1. For the cluster you want to delete, select **Actions** > **Delete**.
+1. Verify details on the confirmation modal and click on **Delete** to confirm the managed cluster deletion.
+
+![delete managed cluster](/img/calico-cloud/managed-clusters-delete.png)
+![delete managed cluster modal](/img/calico-cloud/managed-clusters-delete-step1.png)
+
+:::note
```

Comment:
This note raises questions.

How do I remove logs?

Reply:
It is not possible to remove logs at the moment. The user will be able to view these logs in Kibana